akka-docs/src/main/paradox/additional/osgi.md

```diff
@@ -1,116 +1,6 @@
 # Akka in OSGi
```

Comment:
start a `akka-docs/src/main/paradox/project/migration-guide-2.7.x-2.8.x.md` page and mention the removal there.

I'm not sure we need to keep this osgi.md page at all (possibly redirect to migration guide)?

Reply:
A redirect is better 👍